packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3596,6 +3602,42 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   );
 
+  getLinkHoverTooltip = memoize(
+    (linkHoverTooltipProps: CSSProperties): ReactNode => {
+      if (linkHoverTooltipProps == null) {
+        return null;
+      }
+
+      const { linkHoverDisplayValue } = this.state;
+
+      const wrapperStyle: CSSProperties = {
+        position: 'absolute',
+        ...linkHoverTooltipProps,
+        pointerEvents: 'none',
+      };
+
+      const popperOptions: PopperOptions = {
+        placement: 'bottom',
+      };
+
+      return (
+        <div style={wrapperStyle}>
+          <Tooltip
+            key={Date.now()}
```

Comment:
Did React complain about this not having a key?

Reply:
No, I copy and pasted it from the `cellOverflowTooltip`. Should I remove it?